docs/ReferenceManyToManyField.md

```diff
@@ -7,6 +7,8 @@ title: "The ReferenceManyToManyField Component"
 
 This [Enterprise Edition](https://marmelab.com/ra-enterprise)<img class="icon" src="./img/premium.svg" /> component fetches a list of referenced records by lookup in an associative table, and passes the records down to its child component, which must be an iterator component.
 
+!["ReferenceManyToManyField example with band's performances"](./img/reference-many-to-many-field.png)
```

Comment:
It would be clearer if your screenshot had a title, so that we know it's showing the details of a band. Otherwise, the reader has no idea about what they're looking at.